src/nunit.gui.core/Model/ITestModel.cs

```diff
@@ -78,9 +78,11 @@ public interface ITestModel : IServiceLocator, System.IDisposable
         // See if a test is running
         bool IsTestRunning { get; }
 
+        // Do we have results from running the test?
         bool HasResults { get; }
 
-        IList<RuntimeFramework> AvailableRuntimes { get; }
+        // List of available runtimes
+        IList<IRuntimeFramework> AvailableRuntimes { get; }
```

Comment:
Comment seems unnecessary

Reply:
Fair enough. Trying to be "consistent". :smile_cat: